homeassistant/components/kraken/const.py

```diff
@@ -0,0 +1,28 @@
+"""Constants for the kraken integration."""
+
+DEFAULT_SCAN_INTERVAL = 60
+DEFAULT_TRACKED_ASSET_PAIR = "XBT/USD"
+DISPATCH_CONFIG_UPDATED = "kraken_config_updated"
+
+CONF_TRACKED_ASSET_PAIRS = "tracked_asset_pairs"
+
+DOMAIN = "kraken"
+
+SENSOR_TYPES = [
```

Comment:
Set up some of these sensors to be disabled by default by leveraging the `entity_registry_enabled_default` entity property.

Reply:
Are you saying the number of created sensors is too high and some should be disabled by default to prevent the user being overwhelmed?
This would then mean I decide which information is deemed important enough to be enabled by default?

Reply:
Correct. Users can always enable others again. And don't overwhelm the user. Just stick with the essentials: ask, bid, low_today, high_today.

Reply:
Thank you! Done.